libp2p/protocols/pubsub/pubsub.nim

```diff
@@ -59,6 +61,16 @@ type
     observers: ref seq[PubSubObserver] # ref as in smart_ptr
     msgIdProvider*: MsgIdProvider      # Turn message into message id (not nil)
 
+proc hasPeerID*(t: PeerTable, topic, peerId: string): bool =
```

Comment:
If only used in tests, we should just move it to our test utils; UFCFS is a wonderful thing :)